static/app.css

```diff
@@ -0,0 +1,499 @@
+	input[type="button"]:hover {
+		background-color: #9ee6d0;
+	} 
+
+	.button.full-width,
+	button.full-width,
+	input[type="submit"].full-width,
+	input[type="reset"].full-width,
+	input[type="button"].full-width {
+		width: 100%;
+		padding-left: 0 !important;
+		padding-right: 0 !important;
+		text-align: center; }
+
+	/* Fix for odd Mozilla border & padding issues */
+	button::-moz-focus-inner,
+	input::-moz-focus-inner {
+    border: 0;
+    padding: 0;
+	}
+
+
+/* #Forms
+================================================== */
+
+	form {
+		margin-bottom: 20px; }
+	fieldset {
+		margin-bottom: 20px; }
+	input[type="text"],
+	input[type="password"],
+	input[type="email"],
+	textarea,
+	select {
+		font: 13px "Lato", Helvetica, Arial, sans-serif; 
+		color: #777;
+		margin: 0;
+		max-width: 100%;
+		display: block;
+		background: #fff; }
+	select {
+		padding: 0; }
+	input[type="text"]:focus,
+	input[type="password"]:focus,
+	input[type="email"]:focus,
+	textarea:focus {
+		border: 1px solid #aaa;
+ 		color: #444;
+ 		-moz-box-shadow: 0 0 3px rgba(0,0,0,.2);
+		-webkit-box-shadow: 0 0 3px rgba(0,0,0,.2);
+		box-shadow:  0 0 3px rgba(0,0,0,.2); }
+	textarea {
+		min-height: 60px; }
+	label,
+	legend {
+		display: block;
+		font-weight: bold;  
+	}
+	select {
+		width: 220px; }
+	input[type="checkbox"] {
+		display: inline; }
+	label span,
+	legend span {
+		font-weight: normal;
+		font-size: 13px;
+		color: #444; }
+
+/* #Misc
+================================================== */
+	.remove-bottom { margin-bottom: 0 !important; }
+	.half-bottom { margin-bottom: 10px !important; }
+	.add-bottom { margin-bottom: 20px !important; }
+
+
+div#leaderboard{
+	width:100%; 
+	text-align:center;
+	margin-top:60px; 
+	margin-bottom:60px;
+}
+div#cube {
+	display: none;
+	margin: 10px auto;
+	text-align: center;
+}
+@media (max-width: 768px) {
+	div#leaderboard {
+		display: none;
+	} 
+	div#cube {
+		display: block;
+	}
+}
+div#popsearches div{
+	display:inline;
+}
+div#popsearches {
+	text-align:left;
+	background:#3c9076;
+	padding:20px;  
+	text-transform: uppercase;
+}
+div#popsearches a, div#popsearches, div#popsearches strong{
+	text-decoration: none;
+	color:white;
+}
+div#popsearches a{
+	padding-left:10px; padding-right:10px; border-right:1px solid #ececec;
+}
+div#popsearches a:last-child{
+	border-right:0px none;
+}
+
+.topEarnersWrapper {
+	padding: 5px;
+	color: #3c9076;
+	text-align: center;
+}
+.topEarnersHead {
+	margin: 0 auto;
+	color: inherit;
+	font-weight: bold;
+	text-transform: uppercase;
+}
+.topEarner {
+	display:inline-block;
+	width: 220px;
+	margin: 10px;
+	padding: 5px;
+	border: solid #3c9076;
+	border-width: 0 2px;
+	border-radius: 10px;
+	font-size: 18px;
+	text-transform: uppercase;
+}
+.topEarner:hover {
+	background-color: #9ee6d0;
+}
+.topEarner a {
+	color:inherit;
+	text-decoration: none;
+	cursor: pointer;
+}
+.topEarner p {
+	margin: 0 auto;
+}
+.topEarner p:first-child {
+	font-weight: bold;
+}
+div#share-tools{
```

Comment:
`#` already has ultra-high specificity, so you don't need to add the element name to it. In general, it's best to avoid using element names for non-global styling because you may want to change a div to a section or a span to a b or something but you don't want that to affect your CSS.

Reply:
fixed this